Makefile

```diff
@@ -4,7 +4,7 @@
 CODE_DIRS ?= doc examples examples_flask pyvista tests
 # Files in top level directory
 CODE_FILES ?= *.py *.rst *.md
-CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti,*.vtu,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,./doc/_build/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/examples/*,*.mypy_cache/*,*cover,./tests/tinypages/_build/*,*/_autosummary/*"
+CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti,*.vtu,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,./doc/_build/_static/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/examples/*,*.mypy_cache/*,*cover,./tests/tinypages/_build/*,*/_autosummary/*"
```

Comment:
Why is this necessary? Aren't all things in `_build/` built from checked source?

Reply:
Not quite. There's lots in `_build` that's autogenerated:
```
Running codespell
doc/_build/html/_static/pygments.css:40: nd ==> and, 2nd
doc/_build/html/_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.svg:147: adn ==> and
```

Reply:
@akaszynski according to the diff the original had
```
./doc/_build/*
```
while the new version has 
```
./doc/_build/_static/*
```
This should mean that originally we had "anything in `_build`, and now we have "anything in `_build/static`.

Since this is `CODESPELL_SKIP` these translate to being

> Skip everything in `_build/`

and

> Skip everything in `_build/static`

respectively. Reversing the logic for clarity's sake these are

> Check everything outside `_build`

and

> Check everything outside `_build/static`

respectively.

In other words, the change seems to _newly include_ things in codespell that are in `_build/` but outside `_build/static`. My point was that if everything in `_build` is autogenerated then why do we have to be stricter with the codespell skip?

Is it clearer now?